RecoLocalTracker/SiPixelRecHits/interface/PixelCPEBase.h

```diff
@@ -49,7 +49,8 @@ class PixelCPEBase : public PixelClusterParameterEstimator {
     const PixelTopology* theTopol;
     const RectangularPixelTopology* theRecTopol;
 
-    GeomDetType::SubDetector thePart;
+    //set default value of enum to avoid USBAN errors
+    GeomDetType::SubDetector thePart = GeomDetEnumerators::invalidDet;
```

Comment:
why just this data member and not all the others?

Reply:
It is the only enum variable and the only one causing the UBSAN error (still very strange to me as the uninitialized value is never used). I could add initialization to the other variables if we want

Reply:
ah ok. the usual false positive...